README.md

```diff
@@ -1296,6 +1296,45 @@ _You can enable the following settings in Xcode by running [this script](resourc
   
   </details>
 
+* <a id='extension-access-control'></a>(<a href='#extension-access-control'>link</a>) **Specify the access control for each declaration in an extension individually.** [![SwiftFormat: extensionAccessControl](https://img.shields.io/badge/SwiftFormat-anyObjectProtocol-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md##extensionaccesscontrol)
+
+  <details>
+
+  #### Why?
+
+  Specifying the access control on the declaration itself helps engineers more quickly determine the access control level of an individual declaration.
+
+  ```swift
+  // WRONG
+  public extension Foo {
+    // This declaration doesn't have an explicit access control level.
+    // In all other scopes, this would be an internal function,
+    // but because this is in a public extension, it's actually a public function.
+    func bar() {}
+  }
+
+  // WRONG
+  private extension Foo {
+    func bar() {}
+  }
+
+  // RIGHT
+  extension Foo {
+    // It is immediately obvious that this is a public function,
+    // even if the start of the `extension Foo` scope is off-screen.
+    public func bar() {}
+  }
+
+  // RIGHT
+  extension Foo {
+    // Recall that a private extension actually has fileprivate semantics,
```

Comment:
I think this comment is a bit hard to follow since we're extending `Foo` a bunch of times. I think you are referring to the case where we write a `private` extension on `Foo`. Maybe we can make each paired example extend a unique type for clarity? And maybe have different method names?

It's nice when you could put this code into a playground and it would compile, though not mandatory.